services/explorer-indexer/src/modules/pbft/pbft.entity.ts

```diff
@@ -18,7 +18,7 @@ export class PbftEntity extends BaseEntity implements IPBFT {
   @IsNotEmpty()
   number: number;
 
-  @Column({ nullable: false })
+  @Column({ nullable: false, default: 0 })
```

Comment:
Don't see why we should have default 0 as timestamp. Will there be a case when we will have a pbft block without timestamp and we want to have it 0?

Reply:
Yes, there are data packets where it comes as `undefined`.